### PR TITLE
Auto-update Dashboard

### PR DESCRIPTION
**Pull request by riley**
## What does this PR do?

updates some styles and adds auto-update to the Dashboard.

## How do I test this PR?

- view the dashboard. there should be a little timer at the top that counts down from 5 minutes
- after 5 minutes, the timer should reset and the dashboard reloads

### CODE REVIEW

**Review by davidgljay — APPROVED** [review]
Looks good, though the design feels too strong. It jumps out as the most important thing on the page (color, movement, etc), when it feels like it could be pretty low in the information hierarchy, but that's not a reason to hold up the PR.

**Comment by riley** [comment]
Please wait to merge this one after #396

**Comment by riley** [comment]
agreed @davidgljay I think it's going to be something that always grabs your attention, which might not be good when you're going to be looking at it all day every day.

**Pull request merged**
